client/build.gradle

```diff
@@ -16,6 +16,7 @@ plugins {
 dependencies {
     implementation project(path: ":${rootProject.name}-spi", configuration: 'shadow')
     implementation project(path: ":${rootProject.name}-common", configuration: 'shadow')
+    implementation project(path: ":${rootProject.name}-memory")
```

Comment:
Just curious to know if we should mark this as `shadow`?

Reply:
Memory package doesn't have a publishing task that's why @jngz-es suggested to keep it like this.